src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,7 +66,7 @@ const renderer = ({
       showMore={showMoreConfig !== null}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
-      searchFacetValues={searchForFacetValues && searchForItems}
+      searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

Comment:
It should be undefined if it's not activated. In the current implementation of the refinement list component, the contract is that is there is a function for searchFacetValues, it means that the component should render a search box. If not, no search box.

https://github.com/algolia/instantsearch.js/blob/feat/instantsearch.js/v2/src/components/RefinementList/RefinementList.js#L166-L171